src/components/App/App.js

```diff
@@ -1,48 +1,89 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import classnames from 'classnames'
-import './style.css'
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import classnames from "classnames";
+import "./style.css";
 
-import Notebooks from '../Notes/Notebooks'
+import Notebooks from "../Notes/Notebooks";
```

Comment:
I don't use semicolons in js, so can this prettyfier rule be removed and undone?

Reply:
Yes, certainly. Will do as part of prev comment.